fog/ledger/enclave/src/lib.rs

```diff
@@ -187,6 +190,83 @@ impl LedgerEnclave for LedgerSgxEnclave {
         let outbuf = self.enclave_call(&inbuf)?;
         mc_util_serial::deserialize(&outbuf[..])?
     }
+
+    // Router/store system.
+    fn ledger_store_init(&self, ledger_store_id: ResponderId) -> Result<NonceAuthRequest> {
+        let inbuf = mc_util_serial::serialize(&EnclaveCall::LedgerStoreInit(ledger_store_id))?;
+        let outbuf = self.enclave_call(&inbuf)?;
+        mc_util_serial::deserialize(&outbuf[..])?
+    }
+
+    #[allow(unused_variables)]
```

Comment:
nit: Similar to above, does this need to be on this specific line? If not, remove.

Reply:
Addressed in [278d8c1](https://github.com/mobilecoinfoundation/mobilecoin/pull/2896/commits/278d8c1f98696a964ebd09ae87c4b2d5e316cfd0)